package.json

```diff
@@ -3,24 +3,35 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/babel-plugin": "11.0.0",
+    "@emotion/react": "11.1.1",
+    "@emotion/styled": "11.0.0",
+    "@material-ui/core": "4.11.2",
+    "@material-ui/styles": "4.11.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^10.4.9",
     "@testing-library/user-event": "^12.1.3",
+    "axios": "0.21.0",
+    "dotenv": "^8.2.0",
+    "jest": "24.9.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-icons": "4.1.0",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --updateSnapshot",
+    "test:coverage": "set CI=true && react-scripts test --coverage --updateSnapshot",
```

Comment:
The code coverage report is not working properly 
<img width="472" alt="Screen Shot 2020-12-16 at 17 32 39" src="https://user-images.githubusercontent.com/6569533/102422833-d8e5a280-3fc4-11eb-91cc-de762ef4020e.png">

Reply:
Hi, I don't know why this is happening, but in my computer i could see the coverage, that is why I already sent and do my commits according to the percentage that I found on the terminal?

And also i have to say that I have that same issue on my Mac, so i found that maybe was the OS so i change to my Laptop with windows and there the test run correctly, I add the evidence of my tests that runs perfectly
<img width="1267" alt="test" src="https://user-images.githubusercontent.com/37283578/102426947-fbcc8280-3fd5-11eb-9fe7-e61020d6b079.png">

Reply:
Awesome 🙌, it is weird, but thank you for sharing the final result, I saw the tests and I knew that you got them, I have updated the above note 💯